config/initializers/devise.rb

```diff
@@ -0,0 +1,259 @@
+# Use this hook to configure devise mailer, warden hooks and so forth.
+# Many of these configuration options can be set straight in your model.
+Devise.setup do |config|
+  # The secret key used by Devise. Devise uses this key to generate
+  # random tokens. Changing this key will render invalid all existing
+  # confirmation, reset password and unlock tokens in the database.
+  # config.secret_key = '88359f5e4dfa07da20dade2eb3359abf7d1918a39b54376e0ae1d837cc6c3403c0c21f3b9b9bc0aa914559a440377d19113a8f22132382bb6bf699ff15bfecb9'
+
+  # ==> Mailer Configuration
+  # Configure the e-mail address which will be shown in Devise::Mailer,
+  # note that it will be overwritten if you use your own mailer class
+  # with default "from" parameter.
+  config.mailer_sender = 'please-change-me-at-config-initializers-devise@example.com'
```

Comment:
Is this line left unchanged on purpose? (Since we don't actually have an email adress to use...)